src/bindings/js/node/lib/addon.ts

```diff
@@ -60,6 +60,7 @@ interface Model {
   input(nameOrId?: string | number): Output;
   getName(): string;
   isDynamic(): boolean;
+  get_output_shape(): Output[];
```

Comment:
It's snake_case_notation. We use camelCase on JS side. 
Here you named it right: https://github.com/openvinotoolkit/openvino/blob/dd4802b2b3ca49ca7e9986c1da856ca3634a0c9d/src/bindings/js/node/src/model_wrap.cpp#L23

Reply:
Return type is incorrect, please, check